### ISSUE TITLE
Subs adjacent in time are exported with overlaps

### ISSUE THREAD

**Issue by marcan** [arcusmaximus/YTSubConverter#25]
[Sample video](https://www.youtube.com/watch?v=2VmoSsyXJDw) / [ASS](https://mrcn.st/t/atai_no_saikyou_no_uta_FINAL_v9.wav_en.ass) / [YTT](https://mrcn.st/t/atai_no_saikyou_no_uta_FINAL_v9.wav_en.ytt)

Where the ASS always has lines with adjacent timestamps (look at Default only):

```
Dialogue: 0,0:00:24.19,0:00:27.02,Default,,0,0,0,,{\k17}{\k16}i{\k17}i {\k17}ka{\k50}n{\k16}ji {\k17}no {\k17}bi{\k16}i{\k17}to {\k67}ni
Dialogue: 0,0:00:24.19,0:00:27.02,English,,0,0,0,,and make it into a cool beat
Dialogue: 0,0:00:27.02,0:00:30.02,Default,,0,0,0,,{\k17}{\k33}i{\k50}i {\k17}ka{\k16}n{\k17}ji {\k17}no {\k16}o{\k17}to {\k17}no{\k16}se{\k17}ma{\k17}ku{\k16}t{\k17}te
Dialogue: 0,0:00:27.02,0:00:30.02,English,,0,0,0,,throw in some cool sounds
```

The YTT sometimes ends up with merged lines like this:

```
    <p t="27011" d="9" wp="1" ws="1"><s p="1">ii kanji no biito ni</s><s p="5">
ii kanji no oto nosemakutte</s></p>
    <p t="27011" d="9" wp="1" ws="1"><s p="3">ii kanji no biito ni</s><s p="6">
ii kanji no oto nosemakutte</s></p>
```

And this results in visual glitches. I'm guessing this is a rounding error somewhere? It doesn't seem to happen always, only on some lines, and not in the English translation below nor on the standalone Japanese karaoke subs, but only on the romaji at the top of the English version (all versions should have the exact same sub line timing).

(PS: I released this song yesterday, let me know if you like it ^^)

**Comment by arcusmaximus** [comment]
This indeed looks to be a rounding bug. I found the problem and will fix it in the next release. As a workaround in the meantime, you can apply a \pos tag to all the karaoke lines - this will prevent them from stacking when their times overlap.

(Oh, and nice song by the way!)

**Comment by arcusmaximus** [comment]
The fix is now available in YTSubConverter 1.5.1. Thank you for reporting :)

**Comment by marcan** [comment]
Thanks! I just re-converted and re-uploaded the subs :)
